utils/json-schemas/device_registration.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "https://frinx.io",
+  "$id": "https://frinx.io/device-registration.json",
+  "title": "Device registration",
+  "description": "Device registration schema",
+  "type": "object",
+  "required": ["device_name", "zone_id"],
+  "properties": {
+    "device_name": {
+      "description": "The name of the device.",
+      "type": "string"
+    },
+    "device_size": {
+      "description": "The size of the device.",
+      "type": "string",
+      "enum": ["SMALL", "MEDIUM", "LARGE"]
+    },
+    "device_type": {
+      "description": "The type of the device.",
+      "type": "string"
+    },
+    "device_address": {
+      "description": "The address of the device.",
+      "type": "string"
+    },
+    "device_port": {
+      "description": "The port of the device.",
+      "type": "integer"
```

Comment:
I think we can add some constraint here, valid port range is <0, 65535>

      "minimum": 0,
      "maximum": 65535